pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -351,6 +351,8 @@ CREATE TABLE constraint_db.t1 (
   j "char",
   k VARCHAR(10) COLLATE sv,
   l INT AS (a * b) VIRTUAL,
+  m INT GENERATED ALWAYS AS IDENTITY,
+  n INT GENERATED BY DEFAULT AS IDENTITY,
```

Comment:
@rafiss I added these two columns in `constraint_db.t1`, but ` gmake testlogic TESTCONFIG=fakedist-spec-planning FILES=pg_catalog TESTFLAGS=-rewrite` does not rewrite the file.

Reply:
hm, what if you try without that `TESTCONFIG` variable? (also try with `FILES='pg_catalog|information_schema'` since that file might also need an update)